nsis/src/nsis/NsisLoader.java

```diff
@@ -38,46 +42,40 @@
 import ghidra.util.exception.CancelledException;
 import ghidra.util.task.TaskMonitor;
 import nsis.file.NsisConstants;
+import nsis.file.NsisExecutable;
 import nsis.format.NsisBlockHeader;
 import nsis.format.NsisScriptHeader;
 
-public class NsisLoader extends AbstractLibrarySupportLoader {
-	private long nsis_header_offset = 0;
+public class NsisLoader extends PeLoader {
+	
+	public final static String NE_NAME = "NSIS Executable (NE)";
```

Comment:
I prefer `NSIS Executable (PE + NSIS)`

The acronym PE is widely known, but "NE" is not.

Reply:
Changed